src/shellstrings.c

```diff
@@ -0,0 +1,105 @@
+
+/* 
+ * String output functions to be used in the shell
+ *
+ * See shellstrings.h for function documentation.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "shellstrings.h"
+
+/* See shellstrings.h */
+void shell_intro() {
+    printf("------- Welcome to Spellcheck! ------\n\n");
+
+    printf("Please load a file to begin. Additionally, select an output mode and/or choose dictionary before running the program.\n\n");
```

Comment:
Should be "choose a dictionary", and please delete the second newline.

Reply:
done!